packages/ui/src/app/pages/Forum/components/ForumTabs.tsx

```diff
@@ -3,11 +3,16 @@ import React from 'react'
 import { usePageTabs } from '@/app/hooks/usePageTabs'
 import { Tabs } from '@/common/components/Tabs'
 import { ForumRoutes } from '@/forum/constant'
+import { useMyThreads, UseMyThreadsProps } from '@/forum/hooks/useMyThreads'
+
+const order = { orderKey: 'updatedAt', isDescending: true }
 
 export const ForumTabs = () => {
+  const { pageCount } = useMyThreads({ page: 0, threadsPerPage: 1000, order } as UseMyThreadsProps)
+  const myThreadsCount = pageCount && pageCount > 0 ? pageCount : undefined
```

Comment:
This is counting pages of thread, but there is up to `1000` threads per page

Reply:
how many do you suggest?